docker-compose.yml

```diff
@@ -39,9 +39,10 @@ services:
       METADEFENDER_API_KEY:
     ports:
       - "8000:8000"
-    # Uncomment following to enable debug in development:
-    # stdin_open: true
-    # tty: true
+    # Following allows to execute `docker attach feder_web_1`
+    # necessary when using ipdb in development:
+    stdin_open: true
```

Comment:
I suggest comment out that back. It's out of scope of that PR.

Reply:
I know, it's out of scope but it's just about to be merged into master. I think it's useful to save some time. To be honest, this super correctness makes me work much slower and I don't like the idea of making separate branch just for things so small. I mean, there is nothing unclear in this case, which would lead to any problems in my opinion.

Reply:
Ja skomentowałem, bo odniosłem wrażenie, że zacommitowałeś odkomentowane przypadkiem. Jak wprowadzałeś zacommitowane to się nie odezwałem.

Reply:
Rozumiem, w tym przypadku akurat celowo odkomentowałem, bo pomyślałem, że tak będzie wygodniej dla wszystkich deweloperów. Raczej niczemu to nie szkodzi, bo Dockera mamy tylko dla developmentu, a ułatwia debug. Chyba, że o czymś nie wiem?

Reply:
Jak ułatwia i testy przechodzą to niech będzie.